proposals/utils/validate_proposal.py

```diff
@@ -217,6 +217,13 @@ def _build_forms(proposal: Proposal) -> OrderedDict:
                     session,
                 )
 
+    forms['translated'] = (
+        TranslatedConsentForms,
+        reverse('proposals:translated', args=[proposal.pk]),
+        _('Vertaling Consent Formulieren'),
```

Comment:
I think you forgot to `makemessages` after adding this string because it isn't in `django.po`